beanprice/price.py

```diff
@@ -753,7 +767,7 @@ def process_args():
     setup_cache(args.cache_filename, args.clear_cache)
 
     # Get the list of DatedPrice jobs to get from the arguments.
-    dates = [args.date or None]
+    dates: List[Optional[datetime.date]] = [args.date or None]
```

Comment:
Why this? I don't think it is very useful.

Reply:
This one indeed is not needed, removed.